config/application.yml.default

```diff
@@ -393,6 +394,7 @@ development:
   doc_auth_vendor_randomize_percent: 0
   domain_name: localhost:3000
   enable_rate_limiting: false
+  feature_select_email_to_share_enabled: true
```

Comment:
It's not necessary to override if it's the same as in the top-level scope (the default value).

```suggestion
```